enterprise/check-links.md

```diff
@@ -44,4 +44,7 @@ ephox {
 
 You will also find more Self-hosted setup information on the [Link Checker plugin page]({{ site.baseurl }}/plugins/linkchecker/).
 
-> **Important note:** The Link Checker server currently does not support integration with IBM WebSphere Application Server.
+{% assign pluginname = 'Link Checker' %}
```

Comment:
Page flow doesn't make sense now?

Reply:
Also, this page still has references to the standalone and "Pro" bundle.

> Link Checker is available as a stand-alone plugin or as part of our Pro Bundle. To learn more about your options, check out our pricing and plans here.

Reply:
reorg'ed and removed para about bundles.